www/kanban/inner.js

```diff
@@ -534,6 +536,137 @@ define([
         UI.openCustomModal(editModal.modal);
     };
 
+    addMoveElementButton = function (framework, kanban) {
+        var $container = $(kanban.element);
+        var drag = $('#toggle-drag-off').attr('class').indexOf('toggle-active') !== -1 ? false : true;
+        kanban.options.dragBoards = drag;
+        kanban.options.dragItems = drag;
+        if (drag === false) {
+            if (!kanban) { return; }
+            if (framework.isReadOnly() || framework.isLocked()) { return; }
+
+            var move = function (arr, oldIndex, newIndex) {
+                arr.splice(newIndex, 0, arr.splice(oldIndex, 1)[0]);
+                _updateBoards(framework, kanban, kanban.options.boards, false);
+            };
+
+            var moveBetweenBoards = function (nextBoardItems, elId, boardItems, index, boards, boardId) {
+                nextBoardItems.unshift(elId);
+                boardItems.splice(index, 1);
+                boards.data[boardId].item = boardItems;
+                _updateBoards(framework, kanban, boards, false);
+                $(`.kanban-item[data-eid="${elId}"]`)[0].scrollIntoView();
+            };
+
+            var shiftItem = function (direction, el) {
+                var board = el.parentNode.parentNode.parentNode;
+                var boards = kanban.options.boards;
+                var elId = parseInt($(el).attr("data-eid"));
+                var boardId = parseInt($(board).attr("data-id"));
+                var boardItems = direction === ('up' || 'down') ? kanban.options.boards.data[boardId].item : boards.data[boardId].item;
```

Comment:
This very complex line is the same as: `var boardItems = boards.data[boardId].item;`.

`direction === ('up' || 'down')` this is valid JavaScript but it doesn't really makes sense: `('up' || 'down')` is just `'up'`
And `kanban.options.boards` is exactly the same as `boards` (defined 3 lines above)